### DISCUSSION TITLE
x-for and table

### DISCUSSION THREAD

**Discussion by sqllyw** [alpinejs/alpine#1143] [1. Help] [answered]
Hi,
I got a list that I'd like to render to a table, but it is not visible in <td x-text=??>, but works in <p x-text=??> any idea why?

https://codepen.io/sqllyw/pen/rNWvBRZ

**Comment by ryangjchandler** [comment, marked as answer]
@sqllyw You had a `<div>` inside of a `<table>`, you just need the root `<tr>` - see here: https://codepen.io/ryangjchandler/pen/JjbvjjQ
